src/common/node/childProcess.ts

```diff
@@ -34,13 +34,19 @@ interface ISpawnOptions {
     detached?: boolean;
 }
 
+export interface IExecRejection {
```

Comment:
I'm inclined to change this for a proper Error class which has a stderr property, so we always reject promises with Errors.